lib/role_control/parental_controlled.rb

```diff
@@ -22,7 +22,13 @@ def parent_class
 
       def scope_for(action, user, opts={})
         parent_scope = parent_class.scope_for(action, user, opts)
-        joins(@parent).merge(parent_scope)
+
+        if Panoptes.flipper["no_join_parental_scope"].enabled?
```

Comment:
non-blocking: still think it's kind of confusing to have a negative toggle and it should still be created in the Flipper UI so we've got a list of toggles.